tgui/packages/tgui/interfaces/AccountingConsole.tsx

```diff
@@ -1,132 +1,614 @@
+        <Stack vertical fill>
           <Stack.Item>
-            <Tabs fluid textAlign="center">
-              <Tabs.Tab
-                selected={screenmode === SCREENS.users}
-                onClick={() => setScreenmode(SCREENS.users)}
-              >
-                Users
-              </Tabs.Tab>
-              <Tabs.Tab
-                selected={screenmode === SCREENS.audit}
-                onClick={() => setScreenmode(SCREENS.audit)}
-              >
-                Audit
-              </Tabs.Tab>
-            </Tabs>
+            <Flex height="355px">
+              <Flex.Item width="100px">
+                <Stack mt={1} ml={2}>
+                  <Stack.Item>
+                    <Stack vertical align="center">
+                      <FakeDesktopButton
+                        name="paychecks.exe"
+                        setScreenmode={setScreenmode}
+                        ownerScreenMode={SCREENS.users}
+                      >
+                        <DmIcon
+                          width="70px"
+                          height="70px"
+                          mt={1}
+                          icon="icons/obj/card.dmi"
+                          icon_state="budgetcard"
+                        />
+                      </FakeDesktopButton>
+                      <FakeDesktopButton
+                        name="audit.exe"
+                        setScreenmode={setScreenmode}
+                        ownerScreenMode={SCREENS.audit}
+                      >
+                        <DmIcon
+                          width="70px"
+                          height="70px"
+                          mt={1}
+                          icon="icons/obj/service/bureaucracy.dmi"
+                          icon_state="docs_verified"
+                        />
+                      </FakeDesktopButton>
+                      <FakeDesktopButton
+                        name={ianFileName}
+                        setScreenmode={setScreenmode}
+                        ownerScreenMode={SCREENS.ian}
+                      >
+                        <DmIcon
+                          width="70px"
+                          height="70px"
+                          mt={1}
+                          icon="icons/mob/simple/pets.dmi"
+                          icon_state={young_ian ? 'puppy' : 'corgi'}
+                        />
+                      </FakeDesktopButton>
+                    </Stack>
+                  </Stack.Item>
+                </Stack>
+              </Flex.Item>
+              {screenmode === SCREENS.users && (
+                <Flex.Item grow ml={3}>
+                  <FakeWindow
+                    name="Crew Account Summary"
+                    setScreenmode={setScreenmode}
+                  >
+                    <UsersScreen />
+                  </FakeWindow>
+                </Flex.Item>
+              )}
+              {screenmode === SCREENS.audit && (
+                <Flex.Item grow ml={3}>
```

Comment:
There's a lot of margins in this ui, this makes the layout hacky

Reply:
I don't know how to make things sit correctly otherwise